app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/MainFragment.kt

```diff
@@ -12,16 +12,13 @@ import edu.uc.kovaciad.slicetracker.R
 
 class MainFragment : Fragment() {
 
-    companion object {
-        fun newInstance() = MainFragment()
-    }
-
```

Comment:
Moved companion object to bottom of class to comply with Kotlin best practices: https://kotlinlang.org/docs/coding-conventions.html#class-layout